plugins/ci/pkg/ci/tfsec.go

```diff
@@ -17,37 +17,44 @@ func (ci *CIScan) TerraformEnabled() bool {
 	return *ci.config.Reports.TFSec.Enabled
 }
 
-func (ci *CIScan) ProcessTerraformPaths() (models.ReportInfo, error) {
+func (ci *CIScan) ProcessTerraformPaths() (report models.ReportInfo, areResults bool, err error) {
 	logrus.Infof("processing %d Terraform paths", len(ci.config.Terraform.Paths))
+	if len(ci.config.Terraform.Paths) == 0 {
+		return models.ReportInfo{}, false, nil
+	}
 	var reportProperties models.TFSecReportProperties
 	for _, terraformPath := range ci.config.Terraform.Paths {
 		results, err := ci.ProcessTerraformPath(terraformPath)
 		if err != nil {
-			return models.ReportInfo{}, err
+			return models.ReportInfo{}, false, err
 		}
 		if len(results) > 0 {
 			reportProperties.Items = append(reportProperties.Items, results...)
 		}
 	}
+	if len(reportProperties.Items) == 0 {
+		logrus.Infoln("no Terraform results were returned")
+		return models.ReportInfo{}, false, nil
+	}
 	TFSecVersion, err := commands.Exec("tfsec", "-v")
 	if err != nil {
-		return models.ReportInfo{}, fmt.Errorf("cannot get the version of tfsec: %w", err)
+		return models.ReportInfo{}, false, fmt.Errorf("cannot get the version of tfsec: %w", false, err)
```

Comment:
This looks wrong: `fmt.Errorf("cannot get the version of tfsec: %w", false, err)`